packages/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "../tsconfig.base.json",
+  "files": [],
+  "references": [
```

Comment:
this root tsconfig is what lets us do `tsc -b packages` and have everything build correctly according to their own tsconfigs